x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/siem_auditbeat_ecs/manifest.json

```diff
@@ -23,24 +23,78 @@
   },
   "jobs": [
     {
-      "id": "rare_process_linux_ecs",
-      "file": "rare_process_linux_ecs.json"
+      "id": "rare_process_by_host_linux_ecs",
+      "file": "rare_process_by_host_linux_ecs.json"
     },
     {
       "id": "suspicious_login_activity_ecs",
       "file": "suspicious_login_activity_ecs.json"
+    },
+    {
+      "id": "linux_anomalous_network_activity",
```

Comment:
For consistency with the three earlier jobs, all the job IDs and filenames should end with `_ecs` I think, although this may have an impact on the SIEM code if the jobs are started individually by job ID. @randomuserid are the job IDs referenced directly in the SIEM code?

Reply:
The job ids in the siem modules could be loaded on the fly by calling the `get_module` endpoint. e.g. `api/ml/modules/get_module/siem_auditbeat_ecs`
This might be a way of avoiding keeping a hardcoded list of `configTemplates`.